lib/msf/core/exploit/file_dropper.rb

```diff
@@ -134,7 +166,7 @@ def file_dropper_file_exist?(session, path)
   #
   # @param [String] file The file to delete
   # @return [Boolean] True if the delete command has been executed in the remote machine, otherwise false.
-  def file_dropper_delete(session, file)
+  def file_dropper_delete_file(session, file)
     win_file = file_dropper_win_file(file)
```

Comment:
`file_dropper_win_file` is no longer defined. It has been renamed to `file_dropper_win_path`

Reply:
Oops, I missed one.